fakestorage/object.go

```diff
@@ -107,6 +121,10 @@ func toBackendObjects(objects []Object) []backend.Object {
 			Crc32c:          o.Crc32c,
 			Md5Hash:         o.Md5Hash,
 			ACL:             o.ACL,
+			Created:         getCurrentIfZero(o.Created).Format(time.RFC3339),
```

Comment:
The backend storing dates as RFC3339 strings is something we can changed: I opted for this just to keep it aligned to the gcs API, but its true we are losing precision and we this can be decoupled

Reply:
I think since the API will return RFC3339 anyways, it's fine to lose the precision in the backend.